src/components/GagesBarChartAnimation.vue

```diff
@@ -4859,7 +4866,7 @@
               <text
                 class="cls-61"
                 transform="translate(701.7 1184.5)"
-              >What was the storm surge?</text>
```

Comment:
woa, how did this old surge text get in here? I thought I changed this awhile ago

Reply:
I thought we had changed it, too ?

Reply:
Might be from the infographic changes being pretty manual

Reply:
This one needs a change too `What rivers are at risk?` was something about roads instead